src/frontend/apps/standalone_site/src/features/Playlist/components/PlaylistPage.tsx

```diff
@@ -0,0 +1,156 @@
+import { Box, Button, Heading, Text } from 'grommet';
+import { Spinner } from 'lib-components';
+import { useState } from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+
+import { ReactComponent as CheckListIcon } from 'assets/svg/iko_checklistsvg.svg';
+import { SortableTable } from 'components/SortableTable';
+import { WhiteCard } from 'components/WhiteCard/WhiteCard';
+
+import { usePlaylists } from '../api/usePlaylists';
+
+const messages = defineMessages({
+  title: {
+    defaultMessage: 'My Playlists',
+    description: 'Playlist page title',
+    id: 'features.Playlist.title',
+  },
+  create: {
+    defaultMessage: 'Create playlist',
+    description: 'Create playlist button title',
+    id: 'features.Playlist.create',
+  },
+  noPlaylists: {
+    defaultMessage: 'You have no playlist yet.',
+    description: 'Message displayed when the user has not playlist yet.',
+    id: 'features.Playlist.noPlaylists',
+  },
+  tableTitle: {
+    defaultMessage: 'playlists',
```

Comment:
Would it be possible to use instead
```suggestion
    defaultMessage: ''{item_count, plural, =0 {no playlist} one {# playlist} other {# playlists}}',
```
(event if the `no playlist` is useless) ?